curriculum/challenges/english/25-front-end-development/workshop-fruit-search-app/68245535872ded3cc2900fa8.md

```diff
@@ -0,0 +1,118 @@
+---
+id: 68245535872ded3cc2900fa8
+title: Step 10
+challengeType: 0
+dashedName: step-10
+---
+
+# --description--
+
+Next, pass the `handleSubmit` function to the `onSubmit` prop of the `form` element so the form submission can be handled correctly.
```

Comment:
```suggestion
Pass the `handleSubmit` function to the `onSubmit` prop of the `form` element so the form submission can be handled correctly.
```

In my other suggestion, I used "next" at start of step 9. Pretty minor, but I don't like having that for back-to-back steps - so I thought we could remove it here.